test/vc-data-model-1.0/input/example-8-bad-missing-proof.jsonld

```diff
@@ -5,5 +5,12 @@
   ],
   "id": "urn:uuid:3978344f-8596-4c3a-a978-8fcaba3903c5",
   "type": ["VerifiablePresentation"],
+  "credentialSubject": {
```

Comment:
It is very odd to me to see a verifiable presentation with a credentialSubject. I suppose it is possible to do, but I don't understand why one would do it.

Reply:
Of course, with the view that a verifiable presentation is simply a special type of verifiable credential, and the fact that credentialSubject is a mandatory field, the existence of this construction makes sense, even if it is difficult for me to understand how it will be used. Feels like we've gone and required all verifiable presentations to have a credentialSubject, even if it makes no sense for one to be there.
Maybe the view that a verifiable presentation is simply a special type of verifiable credential is incorrect.

Reply:
Hrm... yeah, this is strange... I think it might be a copy-paste issue?